tools/cr_checker/cr_checker.bzl

```diff
@@ -13,120 +13,72 @@
 
 """Defines Bazel rules for running copyright checks and fixes."""
 
-load("@bazel_skylib//lib:paths.bzl", "paths")
-
-def _cr_impl(ctx):
-    """
-    Implementation function for the copyright rule. This function sets up the
-    necessary command to execute the Python script with the specified mode and source files.
-    Args:
-        ctx (RuleContext): The rule context containing attributes and configuration.
-    Returns:
-        DefaultInfo: Bazel default information provider with the output log file.
-    """
-    script = ctx.attr._script
-    files = []
-
-    files = ctx.files.srcs
-    extensions = ctx.attr.extensions
-
-    text_exts = ""
-    if len(extensions):
-        text_exts = "-e {exts}".format(
-            exts = " ".join([exts for exts in ctx.attr.extensions]),
-        )
-
-    ctx.actions.write(
-        output = ctx.outputs.executable,
-        content = "{tool}.py -t {template} {extensions} {fix} {use_mmap} {debug} {log_file} {offset} {srcs}".format(
-            tool = script.files_to_run.executable.short_path,
-            template = ctx.file.template.path,
-            extensions = text_exts,
-            fix = "--fix" if ctx.attr.fix else "",
-            debug = "--verbose" if ctx.attr.debug else "",
-            use_mmap = "--use_memory_map" if ctx.attr.use_memory_map else "",
-            log_file = "--log-file log.txt" if ctx.attr.log_file else "",
-            offset = "--offset %s" % ctx.attr.offset if ctx.attr.offset else "",
-            srcs = " ".join([f.path for f in files]),
-        ),
-    )
-
-    runfiles = ctx.runfiles(
-        files = files + [ctx.file.template],
-    ).merge(
-        script.default_runfiles,
-    )
-    return [DefaultInfo(runfiles = runfiles)]
-
-_cr_rule = rule(
-    implementation = _cr_impl,
-    executable = True,
-    attrs = {
-        "srcs": attr.label_list(allow_files = True, mandatory = False, default = []),
-        "template": attr.label(allow_single_file = True),
-        "extensions": attr.string_list(mandatory = False, default = []),
-        "debug": attr.bool(mandatory = False, default = False),
-        "fix": attr.bool(mandatory = False, default = False),
-        "offset": attr.int(mandatory = False, default = 0),
-        "log_file": attr.bool(mandatory = False, default = False),
-        "use_memory_map": attr.bool(mandatory = False, default = False),
-        "_script": attr.label(
-            default = Label("//tools/cr_checker/tool:cr_checker"),
-        ),
-    },
-)
-
-def check(
+def checker(
```

Comment:
Can you do cr_checker, or even better copyright_checker.

Reply:
Done.